mdtraj/core/topology.py

```diff
@@ -297,7 +327,8 @@ def to_openmm(self, traj=None):
             angles = traj.unitcell_angles[0]
 
             if np.linalg.norm(angles - 90.0) > 1E-4:
-                raise(ValueError("Unitcell angles must be 90.0 to use in OpenMM topology."))
+                raise(ValueError("Unitcell angles must be 90.0 to use \
+                                 in OpenMM topology."))
 
```

Comment:
Same here.  When breaking strings across lines, use the technique I described above.  This adds unnecessary spaces.  Also, this is no longer true (OpenMM supports triclinics), but I'll do this in a different PR.